### ISSUE TITLE
Show blocked indicator on issues that are blocked by an uncompleted issue

### ISSUE THREAD

**Issue by scotttrinh** [scotttrinh/refined-linear#3]
Acceptance criteria:
If an issue's description contains a special expression indicating that it is blocked by a set of different issues, and if those issues are not completed, show an indicator on the ticket in board view.

**Comment by scotttrinh** [comment]
We'd probably want to enable multiple tickets as blockers, so something like:

```
Blocked by C-1, C-2, C-3
```

Should be good.

It's also easier to copy the link, so perhaps we would need it to be able to parse links instead of just issue IDs?

**Comment by scotttrinh** [comment]
Some nice-to-haves if they're cheap:
- It'd be nice if the indicator was a link
- It'd be nice if hovering the indicator highlighted the blocking issues
- It'd be nice if the indicator was in a stable location
